rebar.config

```diff
@@ -23,6 +23,14 @@
     ]}
 ]}.
 
+{dialyzer, [
+    {warnings, [
+        error_handling,
```

Comment:
Let me know if we want `unhandled_returns`. `unknown` is usually not friendly to `rebar3` plugins.